src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -72,17 +72,6 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 			}
 			.store(in: &subscriptions)
 
-		viewModel.state.$statistics
```

Comment:
Is there another place where the statistics are updated?

Reply:
The statistics data is updated in the HomeState which triggers the update directly inside the statistics cell, there's no need to update the table view around it. This was necessary when we did not show the cell at all if the statistics data was not there yet. But we later changed it to initially show the cell with a height of 0 and then let it update itself internally, so I removed this code.